Header header
int8 point_count
geometry_msgs/Point[] point
